vme/src/db.cpp

```diff
@@ -1752,7 +1752,7 @@ zone_reset_cmd *read_zone(FILE *f, zone_reset_cmd *cmd_list)
 
     while (((cmdno = (ubit8)fgetc(f)) != 255) && !feof(f))
     {
-        CREATE(cmd, zone_reset_cmd, 1);
+        cmd=new zone_reset_cmd();
```

Comment:
It would be really nice if you could use 

 new EMPLACE(zone_reset_cmd) zone_reset_cmd();
 
 Then it'll be scoped under membug if we need to chase something down. The define for EMPLACE() is empty in non-membug mode. So it just disappears unless in memory debug mode.

Reply:
Ah - didn't know that was there. Totally can use it. For next time too - if you hold of the merge I can fix and push up a new commit so that its correct the first time :-)